handle_data.py

```diff
@@ -226,3 +225,64 @@ def manual_check_data(data: dict) -> None:
     for n, column_name in enumerate(list(output_row.columns)):
         print(f"{column_name}: {output_values[0][n]}")
     print("[DEBUG] End of manual data check, you can compare these values with what is in the powerpoint")
+
+<<<<<<< Updated upstream
+
+=======
+>>>>>>> Stashed changes
+def remove_faulty(data: dict, variable):
+    full_data = pd.concat([data["Output"], data["Input"]], axis=1)
+    group = ["INI", "ADA", "VAL"]
+
+    for j in range(len(group)):
+<<<<<<< Updated upstream
+        for i in range(98):
```

Comment:
Now the first 100 scenarios are checked, however, ADA has 200 scenarios. Make it adaptive to the length of the identifier type.